### PR TITLE
[CM_APP_55] Gayatri | Api contract for recover cm id api

### PR DESCRIPTION
**Pull request by gayatrijagtap**

### CODE REVIEW

File: src/main/resources/static/api.yaml [hunk 1]
```diff
@@ -2602,6 +2662,30 @@ components:
...
+          type: string
+          example: Raj Sharma
+        gender:
+          $ref: '#/components/schemas/Gender'
+        phoneNumber:
+          type: string
+        yearOfBirth:
+          type: integer
+          example: 2000
+        pmjayId:
```
Comment (angshu) [inline src/main/resources/static/api.yaml:2678]: I think you need to send as "Identifier" - { "identifierType" : "PMJAYID", "identifierValue" : "ayushman bharat Id value" }

File: src/main/resources/static/api.yaml [hunk 2]
```diff
@@ -2602,6 +2662,30 @@ components:
...
         - gender
+    RecoverCmIdRequest:
+      type: object
+      properties:
+        name:
+          type: string
+          example: Raj Sharma
+        gender:
+          $ref: '#/components/schemas/Gender'
+        phoneNumber:
```
Comment (angshu) [inline src/main/resources/static/api.yaml:2673]: Phone number should also be specified as "Identifier" :  { "identifierType" : "MOBILE", "identifierValue" : "phone number" }

File: src/main/resources/static/api.yaml (hunk 1 above)
Comment (gayatrijagtap) [reply src/main/resources/static/api.yaml]: added "PMJAY ID" to unverified identifiers

File: src/main/resources/static/api.yaml (hunk 2 above)
Comment (gayatrijagtap) [reply src/main/resources/static/api.yaml]: added "Mobile" to verified Identifiers

**Pull request merged**